### ISSUE TITLE
Diskrepanz zwischen Tages-Log und Monats-Log bei PV-Anteilen

### ISSUE THREAD

**Issue by miradarya** [openWB/core#1872]
Die Werte der PV-Energieanteile im monthly und daily log passen nicht zusammen: Die Werte für die Gesamtladung an dem entsprechenden Tag stimmen überein. Die Werte für den PV-Anteil weichen ab. Im Monatslog wird da immer deutlich weniger angegeben als im Tageslog. Dadurch passen auch die angezeigten Werte für den Autarkiegrad nicht.

Details:
[https://forum.openwb.de/viewtopic.php?p=115490#p115490](https://forum.openwb.de/viewtopic.php?p=115490#p115490)
und
[https://forum.openwb.de/viewtopic.php?p=115807#p115807](https://forum.openwb.de/viewtopic.php?p=115807#p115807)

Relevanter Ausschnitt aus dem Debug-Log:
[https://forum.openwb.de/download/file.php?id=25562](https://forum.openwb.de/download/file.php?id=25562)

Der Fehler existiert bereits seit Längerem und tritt aktuell auch mit Release 2.1.5 auf.

**Comment by miradarya** [comment]
Der Fehler besteht mit 2.1.7 Beta 2 weiterhin. Aktuelle Log-Daten aus Januar: [https://forum.openwb.de/download/file.php?id=27635](https://forum.openwb.de/download/file.php?id=27635)

**Comment by kevinwieland** [comment]
Ist das Problem aktuell noch nachstellbar?

**Comment by miradarya** [comment]
Ja, das Verhalten ist unverändert. Die Diagramm-Auswertung für März (monatlich) zeigt für "Ladung (PV-Anteil)" einen Wert von 147,301 kWh. Die Summe der entsprechenden täglichen Werte vom 1.3. bis zum 31.3. beträgt jedoch 155,964 kWh.
